src/ffi.md

```diff
@@ -705,8 +705,14 @@ for more information.
 # Representing opaque structs
 
 Sometimes, a C library wants to provide a pointer to something, but not let you
-know the internal details of the thing it wants. The simplest way is to use a
-`void *` argument:
+know the internal details of the thing it wants. The simplest way is to use "extern types".
+But it's currently (as of June 2021) unstable and has some unresolved questions,
+see the [RFC page][extern-type-rfc] and the [tracking issue][extern-type-issue] for more details.
```

Comment:
Hmm, instead of starting off with talking about something that is unstable and likely something people can't use, what do you think about moving into a side note?  That is, take what you have written here, and move it down to the bottom of the page and make it a blockquote.

Reply:
Sounds good! Addressed via 538345a.